content/en/docs/how-tos/use-registries-in-build-farm.md

```diff
@@ -0,0 +1,118 @@
+{
+	"auths": {
+		"registry.ci.openshift.org": {
+			"auth": "token"
+		}
+	}
+}
+```
+
+# Get token for a service account
+
+If some robot is required to log into the central CI registry, we need to create a service account: Choose a new namespace, and create a pull request to add a folder with the same name in [release/clusters/app.ci](https://github.com/openshift/release/tree/master/clusters/app.ci) containing the yaml files of the namespace, the service account, and its RBACs. The following `rolebinding` allows `sa/image-puller` in `namespace/ci` to pull all images in the namespace.
+
+```yaml
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: interop-qe-image-puller-binding
+  namespace: ci
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:image-puller
+subjects:
+- kind: ServiceAccount
+  namespace: ci
+  name: image-puller
+```
+
+After the pull request is merged, the yaml files will be applied automatically to `app.ci`. We can then get the token of, e.g., `SA/image-puller` in `namespace/ci`:
+
+```bash
+$ namespace=ci
+$ sa=image-puller
+$ hostname=image-registry.openshift-image-registry.svc:5000
+$ oc get secrets --namespace $namespace -o json | jq --arg sa $sa '.items[] | select(.type=="kubernetes.io/dockercfg") | select(.metadata.annotations["kubernetes.io/service-account.name"]==$sa) | .data[".dockercfg"]' --raw-output | base64 --decode | jq --arg h $hostname 'with_entries(select(.key==$h))'
+{
+  "image-registry.openshift-image-registry.svc:5000": {
+    "auth": "token"
+  }
+}
+```
+
+The token can be used to modify the `auth.json` file above.
+
+# Access the images in CI
+
+Determine which cluster the job is running on, e.g. [the follow line](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/branch-ci-opendatahub-io-odh-manifests-master-images/1329789689988648960#1:build-log.txt%3A8) in the log of job `branch-ci-opendatahub-io-odh-manifests-master-images` says that the job runs on `build02` and that `ci-op-2c2tvgti` is the temporary namespace for the test:
+
+```
+2020/11/20 14:12:28 Using namespace https://console.build02.ci.openshift.org/k8s/cluster/projects/ci-op-2c2tvgti
+```
+
+All the images used in test are stored in the imagestreams in `ci-op-2c2tvgti` which is also available as an artifact of the job: [imagestreams.json](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/branch-ci-opendatahub-io-odh-manifests-master-images/1329789689988648960/artifacts/build-resources/imagestreams.json).
+
+The image for `istag/pipeline:odh-manifests-tests` in `namespace/ci-op-2c2tvgti` can be pulled by:
+
+```bash
+$ podman pull --authfile auth.json  registry.build02.ci.openshift.org/ci-op-2c2tvgti/pipeline:odh-manifests-tests
+```
+
+Note that the `auth.json` file has to contain a valid token for `registry.build02.ci.openshift.org` from a user or a service account which has permissions to pull the image.
+
+# Access the promoted images
```

Comment:
Accessing the promoted image only differs from accessing any other image by understanding how promotion works for which we have dedicated docs, so not sure if this is helpful here

Reply:
Added the link to the promtion doc.

This is for https://issues.redhat.com/browse/DPTP-1613?focusedCommentId=15512622&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-15512622